recipes/localstack-ext/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "localstack-ext" %}
+{% set version = "0.10.24" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: bb7d3953e45af2df720b1327ce0c12d20b88fd03a63b0291ca9a132c748eb628
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
```

Could it be ``noarch``?
```suggestion
  script: "{{ PYTHON }} -m pip install . -vv"
  noarch: python
```

---

Please see
https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python